lib/service/connect.go

```diff
@@ -184,7 +184,8 @@ func (process *TeleportProcess) connect(role types.SystemRole, opts ...certOptio
 		}
 		// no state recorded - this is the first connect
 		// process will try to connect with the security token.
-		return process.firstTimeConnect(role)
+		c, err := process.firstTimeConnect(role)
+		return c, trace.Wrap(err)
```

Comment:
That was essentially a missing err check, right? [I ran into a similar issue](https://gravitational.slack.com/archives/C0DF0TPMY/p1688395628825499) a couple of months ago. IIRC, turning on the linter back then was a somewhat large ordeal because there are cases where skipping an error check is (at least it seems) valid, like `Close` calls in defers.

Reply:
Well it's not a missing error check, as we were returning the error.

By not wrapping it, we were just skipping a level in the stack trace.

Reply:
Ah, makes sense, I forgot that `process.firstTimeConnect` was still returning two values and not just one.